### PR TITLE
Feature/il 2331

### PR DESCRIPTION
**Pull request by alvaro-ipor**

### CODE REVIEW

File: test/IporSwapLogic/SwapLogicCalculateQuasiInterest.t.sol [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+        assertEq(quasiIFixed, 3122249099904000000000000000000000000000000000000);
+        assertEq(quasiIFloating, 31126978080000000000000000000000000000000000000);
+    }
+
+    function testShouldRevertWhenClosingTimestampIsLowerThanOpenTimestamp() public {
+        // given
+        IporTypes.IporSwapMemory memory swap = IporTypes.IporSwapMemory(
+            TestConstants.ZERO, // id
+            _admin, // buyer
+            block.timestamp + TestConstants.PERIOD_50_DAYS_IN_SECONDS, // openTimestamp
```
Comment (mario-ipor) [inline test/IporSwapLogic/SwapLogicCalculateQuasiInterest.t.sol:96]: Remove above TestConstants.PERIOD_50_DAYS_IN_SECONDS

File: test/IporSwapLogic/SwapLogicCalculateQuasiInterest.t.sol [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+            block.timestamp + TestConstants.PERIOD_75_DAYS_IN_SECONDS, // closeTimestamp
```
Comment (mario-ipor) [inline test/IporSwapLogic/SwapLogicCalculateQuasiInterest.t.sol:97]: Use block.timestamp + TestConstants.SWAP_DEFAULT_PERIOD_IN_SECONDS

File: test/IporSwapLogic/SwapLogicCalculateQuasiInterest.t.sol [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+            9870300000000000000000 * 10, // notional
+            987030000000000000000, // ibtQuantity
+            TestConstants.PERCENTAGE_4_18DEC, // fixedInterestRate
+            TestConstants.TC_LIQUIDATION_DEPOSIT_AMOUNT_18DEC, // liquidationDepositAmount
+            1 // state
+        );
+        // when
+        vm.expectRevert("IPOR_319");
+        _iporSwapLogic.calculateQuasiInterest(
+            swap, swap.openTimestamp - TestConstants.SWAP_DEFAULT_PERIOD_IN_SECONDS, 1 * TestConstants.D18
```
Comment (mario-ipor) [inline test/IporSwapLogic/SwapLogicCalculateQuasiInterest.t.sol:109]: Should work swap.openTimestamp - any value.

**Review by mario-ipor — CHANGES_REQUESTED** [review]

File: test/IporSwapLogic/SwapLogicCalculateQuasiInterest.t.sol (hunk 3 above)
Comment (alvaro-ipor) [reply test/IporSwapLogic/SwapLogicCalculateQuasiInterest.t.sol:109]: Do I leave as is? Or change with something else?

**Pull request merged**